app/lib/services/restApi.ts

```diff
@@ -681,7 +681,7 @@ export const getSyncThreadsList = ({ rid, updatedSince }: { rid: string; updated
 		updatedSince
 	});
 
-export const runSlashCommand = (command: string, roomId: string, params: string, triggerId?: string, tmid?: string) =>
+export const runSlashCommand = (command: string, roomId: string, params: string, triggerId?: string, tmid?: string | null) =>
 	// RC 0.60.2
 	sdk.post('commands.run', {
```

Comment:
IMHO, I think the best approach is change `tmid` @691 to `...(tmid && { tmid })`, also we can remove the `null`